.azure-pipelines/generation-templates/objc.yml

```
@@ -5,3 +5,9 @@ steps:
     BuildConfiguration: $(buildConfiguration)
     OutputFullPath: $(typewriterDirectory)/output
     RepoModelsDir: $(Build.SourcesDirectory)/msgraph-sdk-objc-models/GeneratedModels/
+- pwsh: |
+      gem install xcodeproj
+      & chmod +x .\add_generated_models.rb
+      ruby .\add_generated_models.rb | Write-Host
```

forward slash for unix environment.
```suggestion
      & chmod +x ./add_generated_models.rb
      ruby ./add_generated_models.rb | Write-Host
```

---

Thank you!. I have that change locally. I'm hitting a perm issue installing the gem.